rxbinding-design/build.gradle

```diff
@@ -42,6 +42,7 @@ android {
   lintOptions {
     textReport true
     textOutput 'stdout'
+    disable 'ResourceType', 'PrivateResource'
```

Comment:
Updating the Android Gradle Plugin caused those changes, I assume. They updated and improved lint there.